Meta/Lagom/CMakeLists.txt

```diff
@@ -418,6 +418,28 @@ if (BUILD_LAGOM_TOOLS)
 endif()
 
 if (BUILD_LAGOM)
+
+    # Create lagom equivalent of Serenity's Build/foo/Root/res (which for Serenity becomes /res in the disk image).
+    # FIXME: Add more files as needed.
+    set(LAGOM_RES "${Lagom_BINARY_DIR}/Root/res")
```

Comment:
hm. I wonder if `share/lagom` would be more friendly? Not really that interesting of a distinction without install rule changes tho. So, lgtm.